java/sage/Wizard.java

```diff
@@ -3115,13 +3116,10 @@ public boolean addSeriesInfoPublic(int legacySeriesID, String title, String netw
     return addSeriesInfo(legacySeriesID, title, network, description, history, premiereDate, finaleDate, airDOW, airHrMin, imageURL, people, characters) != null;
   }
   public SeriesInfo addSeriesInfo(int legacySeriesID, int showcardID, String title, String network, String description, String history, String premiereDate,
-      String finaleDate, String airDOW, String airHrMin, String[] people, String[] characters, byte[][] imageURLs)
+      String finaleDate, String airDOW, String airHrMin, Person[] people, String[] characters, byte[][] imageURLs)
```

Comment:
I understand, at last in the case, I'm fairly certain I put that one there, but I will fix that and note for future reference.